daemon/containerd/image_tag.go

```diff
@@ -73,6 +73,17 @@ func (i *ImageService) TagImage(ctx context.Context, imageID image.ID, newTag re
 		return nil
 	}
 
+	builderLabel, ok := sourceDanglingImg.Labels[imageLabelClassicBuilderParent]
+	if ok {
```

Comment:
Not a blocker, but it's a common pattern to inline these (when `ok` vars are involved); also in some cases this helps making it clear a variable is locally-scoped to the condition ("look no further; it's only used here").

```go
	if builderLabel, ok := sourceDanglingImg.Labels[imageLabelClassicBuilderParent]; ok {
```